howto/use-w3c-tracecontext/README.md

```diff
@@ -0,0 +1,243 @@
+# W3C Trace Context - Background
+
+Distributed tracing is a methodology implemented by tracing tools to follow, analyze and debug a transaction across multiple software components. Typically, a distributed trace traverses more than one component which requires it to be uniquely identifiable across all participating systems. Trace context propagation passes along this unique identification. Today, trace context propagation is implemented individually by each tracing vendor. In multi-vendor environments, this causes interoperability problems, like:
+
+Traces that are collected by different tracing vendors cannot be correlated as there is no shared unique identifier.
+Traces that cross boundaries between different tracing vendors can not be propagated as there is no uniformly agreed set of identification that is forwarded.
+Vendor specific metadata might be dropped by intermediaries.
+Cloud platform vendors, intermediaries and service providers, cannot guarantee to support trace context propagation as there is no standard to follow.
+In the past, these problems did not have a significant impact as most applications were monitored by a single tracing vendor and stayed within the boundaries of a single platform provider. Today, an increasing number of applications are highly distributed and leverage multiple middleware services and cloud platforms.
+
+This transformation of modern applications calls for a distributed tracing context propagation standard.
+
+W3C trace context specification defines a universally agreed-upon format for the exchange of trace context propagation data - referred to as trace context. Trace context solves the problems described above by
```

Comment:
The link to the context (which is provided further down) might be better placed on the words "W3C trace context" here